src/types/client-manager.ts

```diff
@@ -92,6 +92,11 @@ export class ClientManager {
     return this._indexer
   }
 
+  /** Returns an algosdk Indexer API client or `undefined` if it's not been provided. */
+  public get indexerIfPresent(): algosdk.Indexer | undefined {
```

Comment:
Wondering if we should add this for kmd as well?

Reply:
Could do, will leave for now, it felt less useful